README.md

```diff
@@ -195,12 +195,55 @@ Executed using lambdas created for deletion purposes, once the deletion is succe
 
 ## Contributing to Trapheus
 
+Prepare your environment. Install tools as needed.
+
+* [Git Bash](https://gitforwindows.org/) used to run Git from Command Line.
+* [Github Desktop](https://desktop.github.com/) Git desktop tool for managing pull requests, branches and repos.
+* [Visual Studio Code](https://code.visualstudio.com/) Full visual editor. Extensions for Github can be added.
+* Or an editor of your choice.
+
+
 1. Fork Trapheus repo
-2. Make changes locally and add unit tests as needed
-3. Run the test suite in the repo to ensure existing flows are not breaking
-```
-cd Trapheus
-python -m pytest tests/ -v #to execute the complete test suite
-python -m pytest tests/unit/test_get_dbstatus_function.py -v #to execute any individual test
-```
-4. Raise a pull request from the fork
+1. Create a working branch.
```

Comment:
Nit: please correct the numbering on the steps

Reply:
@namitad has this been resolved? The step that was added is out of order.

Reply:
@cawrites all the steps added have the numbering 1. please rectify that in your PR

Reply:
This is what's showing in the code already:
1. Fork Trapheus repo
1. Create a working branch.
    ```bash
    git branch trapheus-change1
    ```
1. Confirm working branch for changes.
   ```bash
    git checkout trapheus-change1
   ```

Reply:
The PR currently shows:
1. Fork Trapheus repo
1. Create a working branch.
    ```bash
    git branch trapheus-change1
    ```
1. Confirm working branch for changes.
   ```bash
    git checkout trapheus-change1
   ```
You can combine boths command by typing `git checkout -b trapheus-change1`.
1. Make changes locally using an editor and add unit tests as needed.
1. Run the test suite in the repo to ensure existing flows are not breaking.
    ```bash
       cd Trapheus
       python -m pytest tests/ -v #to execute the complete test suite
       python -m pytest tests/unit/test_get_dbstatus_function.py -v #to execute any individual test
    ```
1. Stage edited files.
   ```bash
      git add contentfile.md 
    ```
    Or use `git add . ` for multiple files.

1. Commit changes from staging.
    ```bash
       git commit -m "trapheus-change1"
    ```
1. Push new changes to github
    ```bash
       git push --set-upstream origin trapheus-change1
    ```
1. Verify status of branch
    ```bash
    git status
    ```
    Review `Output` to confirm commit status.


1. Git push
    ```bash
        git push --set-upstream origin trapheus-change1
    ```

1. The `Output` will provide a link to create your Pull Request.

Reply:
![image](https://user-images.githubusercontent.com/54048339/94847324-4bd47980-03d7-11eb-88ea-aca74e38424a.png)
I am not finding the reported issue.

Reply:
@namitad @stationeros please see Code Review page.